test/fw/ptl/utils/pbs_testsuite.py

```diff
@@ -561,8 +497,8 @@ def check_users_exist(cls):
         Check whether the user is exist or not
         """
         testusersexist = True
-        for u in [TEST_USER, TEST_USER1, TEST_USER2, TEST_USER3]:
-            rv = cls.du.check_user_exists(str(u))
+        for u in PBS_ALL_USERS:
```

Comment:
Do we have to check for all users? I don't recommend changing this to PBS_ALL_USERS. REQUIRED_USERS is enough.

Reply:
@vstumpf I don't agree with you here because as per PTL all users should exist. If you think that all users/groups defined in PTL is not required then we should remove it completely from PTL (not just this check)... But changing users/groups in PTL will be separate stuff than what's this PR is doing...

We define users/groups but we don't check them then what's point of defining them?

Reply:
I agree the changes of users/groups should be in a different PR. For now, we should put this back to the original 4 users.

Reply:
> I agree the changes of users/groups should be in a different PR.

Thank you.

> For now, we should put this back to the original 4 users.

I don't agree with this as previously PTL was doing the wrong thing by checking only 4 users and so we changed since we are changing the respective area in code.
We should change back to the previous 4 users check only if we are facing any problem which checking all users (which you will if you are creating proper users and groups before running PTL).
@vstumpf Saying above leads me to ask you a question, why are you insisting to put back check for only 4 users? Are you facing any problem having a check for all users?

Reply:
@vstumpf : We might need only few users for test cases from but in overall framework we need all the required users to exists . So i agree with @hirenvadalia and @shilpakodli  here. 

Lets fix it instead of postponing.